Intersect.Client.Core/MonoGame/Input/MonoInput.cs

```diff
@@ -237,6 +237,9 @@ private void CheckMouseButton(Keys modifier, ButtonState bs, MouseButton mb)
         }
     }
 
+    private const double ZoomDelayInMilliseconds = 300;
```

Comment:
Change this to `private static readonly TimeSpan ZoomDelay = TimeSpan.FromMilliseconds(300);`